dcos_installer/test_backend.py

```diff
@@ -67,6 +67,35 @@ def test_set_superuser_password(tmpdir):
         assert passlib.hash.sha512_crypt.verify('foo', config['superuser_password_hash'])
 
 
+def test_generate_node_upgrade_script(tmpdir):
+    upgrade_config = """
+---
+# The name of your DC/OS cluster. Visable in the DC/OS user interface.
+cluster_name: 'DC/OS'
+master_discovery: static
+exhibitor_storage_backend: 'static'
+resolvers:
+- 8.8.8.8
+- 8.8.4.4
+ssh_port: 22
+process_timeout: 10000
+bootstrap_url: file:///opt/dcos_install_tmp
+master_list: ['10.0.0.1', '10.0.0.2', '10.0.0.5']
+"""
+
+    with tmpdir.as_cwd():
+        create_fake_build_artifacts(upgrade_config, tmpdir)
```

Comment:
it makes sure we are building the artifacts in the tmpdir, it was needed when I was creating files here, will get rid of it , the create artifacts takes care of it